contracts/cw-core/src/tests.rs

```diff
@@ -2199,10 +2214,31 @@ fn test_migrate() {
     assert_eq!(new_state, state);
 }
 
+#[test]
+fn test_migrate_from_beta() {
```

Comment:
Might be nice to write a version of this test using the integration test approach used by `test_migrate_from_compatible`. IMO, those tests tend to cover a little more / make us a little more resistant to weird regressions later (fewer builtin assumptions about state).